style.css

```diff
@@ -0,0 +1,521 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Grape+Nuts&family=Roboto:ital,wght@0,100;1,100;1,300;1,400&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'poppins', sans-serif;
+}
+
+body a {
+  text-decoration: none;
+  color: #344563;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.header nav ul {
+  display: flex;
+  justify-content: space-around;
+}
+
+.header nav ul li {
+  list-style: none;
+  padding: 10px;
+}
+
+.header nav ul li a {
+  text-decoration: none;
+  color: #344563;
+  font-size: 15px;
+  line-height: 20px;
+  text-align: center;
+}
+
+.logo-text {
+  color: #6070ff;
+  font-size: 18px;
+  line-height: 20px;
+  text-align: center;
+  vertical-align: center;
+  font-weight: 700;
+}
+
+.header-section {
+  min-height: 100vh;
+  background-image: url(images/Header\ bg@2x.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position-y: 20px;
+  max-width: 100vw;
+}
+
+.about-section {
+  height: 100vh;
+  margin: auto;
+  width: 90%;
+  margin-top: 15%;
+  margin-right: 40px;
+}
+
```

Comment:
- [ ] The page is divided half of it contains the project's picture and the other half contains the project's description, these switch places with each project (picture is on the left and description on the right for the first project, description on the left and picture on the right for the second project and so on and so forth). and they must be design horizontally, Kindly fix this and refer yourself on Figma design 😉  

Your Design 👇 
![big](https://user-images.githubusercontent.com/88894525/192996526-22792136-6ec3-4b16-84c2-01760f90a445.png)

Figma Design 👇 
![top](https://user-images.githubusercontent.com/88894525/192999765-583a7327-cd09-4837-9cec-ba729ec76d9f.png)

Reply:
![image](https://user-images.githubusercontent.com/41564330/193005044-b73d0c70-dd45-4790-a31d-3771323d390a.png)
@Alfredbis29